### DISCUSSION TITLE
Choose 'from'/sender?

### DISCUSSION THREAD

**Discussion by francoism90** [spatie/laravel-database-mail-templates#43] [Q&A]
Is this possible? It seems to be missing.

Sometimes one want to send with `info@example.com` or `noreply@example.com`.

**Comment by freekmurze** [comment]
I think you could change the default `from` in the `mail` config file.

**Reply by francoism90** [reply]
@freekmurze Thanks for your reply. :)

I was referring to this: https://laravel.com/docs/9.x/mail#using-the-from-method

Not a deal(breaker).
